transport/src/main/java/io/el/channel/ChannelSingleThreadEventLoop.java

```diff
@@ -0,0 +1,62 @@
+package io.el.channel;
+
+import io.el.concurrent.EventLoop;
+import io.el.concurrent.SingleThreadEventLoop;
+import io.el.internal.ObjectUtil;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.Set;
+import java.util.concurrent.Executor;
+
+public class ChannelSingleThreadEventLoop extends SingleThreadEventLoop implements
```

Comment:
Please change this class to an abstract class. The concrete way of how to `run()` should be implemented child class